src/client/src/reducers/dependencyInfoReducers/index.ts

```diff
@@ -0,0 +1,25 @@
+import { combineReducers } from "redux";
+import { WIZARD_INFO_TYPEKEYS } from "../../actions/wizardInfoActions/typeKeys";
+
+export const initialState = {};
+
+export const updateDependencyInfo = (
+  state: any = initialState,
+  action: any
```

Comment:
Would it be possible to make this slightly more type safe (See how we do it with other reducers). (State + Action).